seed/utils/api.py

```diff
@@ -167,7 +167,7 @@ def get_api_request_user(request):
     """
     Determines if this is an API request and returns the corresponding user if so.
     """
-    if request.is_ajax():
+    if request.headers.get('x-requested-with') == 'XMLHttpRequest':
```

Comment:
I don't think there's any meaningful difference here, but how would you feel about switching it to the way they had it implemented internally?
https://github.com/django/django/blob/03b733d8a8f5b5d8ca0a3cc32b7e6e6c046d95e2/django/http/request.py#L232

Reply:
like this?
```
request.META.get('HTTP_X_REQUESTED_WITH') == 'XMLHttpRequest'
```
we could, but the [docs](https://docs.djangoproject.com/en/3.2/releases/3.1/#id2) recommend the current way. 
```
If you are writing your own AJAX detection method, request.is_ajax() can be reproduced exactly as request.headers.get('x-requested-with') == 'XMLHttpRequest'.
```

Reply:
oh shoot, I missed that, let's do that instead!